chips/apollo3/src/gpio.rs

```diff
@@ -155,6 +157,77 @@ impl Port<'_> {
         }
     }
 
+    pub fn enable_sx1262_radio_pins(&self) {
```

Comment:
This is not ideal, but it seems like the GPIO controller on this chip is so weird that it kind of forces code like this.

Can you at least comment this function explaining that it is board-specific and would ideally be made generic in the future?

Reply:
Yeah, the GPIO controller is very flexible, which also means we need to configure it for every use case.

Eventually we can collapse this all into a generic function for this chip (which is what the HAL does) but for the time being I find these explicit functions simpler to follow

Reply:
I added a comment though :)